install/kubernetes/helm/istio/files/injection-template.yaml

```
@@ -52,14 +51,21 @@ initContainers:
   securityContext:
   {{- if not .Values.istio_cni.enabled }}
     allowPrivilegeEscalation: {{ .Values.global.proxy.privileged }}
+    privileged: {{ .Values.global.proxy.privileged }}
+  {{- else }}
+    allowPrivilegeEscalation: false
+    privileged: false
+  {{- end  }}
```

There's no reason to ignore the `.Values.global.proxy.privileged` user setting when CNI is enabled.
Revert that change:
```suggestion
    allowPrivilegeEscalation: {{ .Values.global.proxy.privileged }}
    privileged: {{ .Values.global.proxy.privileged }}
```

---

Get it. The privilege is safe since it run iptables binary only. What's more it reduced the changes when we need to inject debug util. Will revert.

---

fixed. Thank you for the suggestion. I didn't just accept it because I thought golden files also needs update...